packages/node_modules/@ciscospark/widget-recents/README.md

```diff
@@ -75,6 +76,32 @@ For the latest builds that are pulled from the head of the master branch:
   npm run build:package widget-recents
   ```
 
+### NPM
+
+#### **NOTE: NPM INSTALL IS CURRENTLY IN BETA AND NOT SUPPORTED**
+
+To use the space widget within an existing React appliction, a developer can install the component directly from npm.
+
+1. Install the module via NPM
+
+  ```bash
+  npm install --save @ciscospark/widget-recents
+  ```
+
+2. Add the following import statements
+
+  ```js
+  import RecentsWidget from '@ciscospark/widget-recents';
+  // Sass import required for styling widgets
+  import '@ciscospark/widget-recents/src/momentum.scss';
+  ```
+
+3. Finally, render the widget
+
+  ```js
```

Comment:
we are not going to share other props?

Reply:
All of the configuration properties are listed in the `configuration` section. I'll add a link to them so we don't duplicate them.